spec/integration/suite_hooks_errors_spec.rb

```diff
@@ -86,6 +87,7 @@ def run_spec_expecting_non_zero(before_or_after)
   end
 
   it 'nicely formats errors from multiple :suite hooks of both types and exits with non-zero' do
+    write_file ".rspec", "--no-profile"
```

Comment:
Maybe for aruba we should have a step that rights a full "standard" configuration

Reply:
Fair enough. Actually, I found an `isolated_home` metadata helper that does that.